pulsar-client-cpp/python/pulsar/schema/definition.py

```diff
@@ -393,6 +413,9 @@ def python_type(self):
         return list
 
     def validate_type(self, name, val):
+        if val is None:
+            return None
```

Comment:
Should return the `self.default()`, not `None` automatically

Reply:
Good catch, I'll fix this. Thanks